content/cookbook/development/using-mainnet-accounts-programs.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Using Mainnet Accounts and Programs
+sidebarSortOrder: 5
+description:
+  Learn how to use the Mainnet accounts and programs in your local development
+  environment.
+---
+
+Oftentimes, local tests rely on programs and accounts available only on mainnet.
+The Solana CLI allows to both:
+
+- Download Programs and Accounts
+- Load Programs and Accounts to a local validator
+
+### How to load accounts from mainnet
+
+It is possible to download the SRM token mint account to file:
```

Comment:
Project Serum (`SRM`) doesn't exist anymore. Pick a different token mint! Maybe Openbook DEX as that's Serum's replacement.